data_loading.py

```diff
@@ -145,7 +145,7 @@ def load_supernatural_instructions_task(args):
     """
 
     # SuperNaturalInstructions Tasks without a defined format
-    if any(t in args.task_filename for t in SUPERNATURAL_INSTRUCTIONS_TASKS_WITH_NO_FORMAT):
+    if any(t in args.task_filename for t in SUPERNATURAL_INSTRUCTIONS_TASKS_WITH_NO_FORMAT + OPEN_GENERATION_SUPERNATURAL_INSTRUCTIONS_TASKS):
```

Comment:
Seems like `_setup_non_formatted_dataset_with_one_field_only` works well for generation tasks out of the box.